### PR TITLE
Kommander 1.2 GA Docs

### PR DESCRIPTION
**Pull request by gracedo**
## Jira Ticket
Before creating this pull request, make sure you have a separate ticket for the docs team to track their work. If you need assistance, ask in the #documentation Slack channel.

## Description of changes being made
In preparation for Kommander 1.2 GA, remove `beta` annotations from all docs.

## Checklist
- [x] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [x] Create your PR against `staging`, not `master`. 
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> ie `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

File: pages/ksphere/kommander/1.2/ldap/index.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ---
 layout: layout.pug
-beta: true
+beta: false
 navigationTitle: LDAP Directory
 title: External LDAP Identity Provider
 menuWeight: 10
 excerpt: Use your LDAP Identity Provider with Konvoy and Kommander
-beta: true
+beta: false
```
Comment (msdolbey) [inline pages/ksphere/kommander/1.2/ldap/index.md:8]: @gracedo or @cmastr 
Let's take out this second one - it kind of gets wacky if it has two tags like this.

```suggestion
```

**Review by msdolbey — APPROVED** [review]
👍 
Nice

**Review by msdolbey — APPROVED** [review]
👍 
Nice

**Review by cmastr — APPROVED** [review]

**Comment by juliangieseke** [comment]
RC3 just got tagged with some added features - judging from the version bumps
https://github.com/mesosphere/kubeaddons-kommander/releases/tag/v1.2.0-rc.3
this PR probably needs to be updated accordingly.

**Comment by cmastr** [comment]
> RC3 just got tagged with some added features - judging from the version bumps
> https://github.com/mesosphere/kubeaddons-kommander/releases/tag/v1.2.0-rc.3
> this PR probably needs to be updated accordingly.

@juliangieseke done. Components needed updating, but that was all.

**Comment by cmastr** [comment]
Closing this PR. GA has been pushed out a week so will have to update release notes closer to GA and remove beta tags.

**Pull request closed without merging**